hoot-core/src/main/cpp/hoot/core/ops/ReplaceRoundabouts.cpp

```diff
@@ -59,13 +61,17 @@ void ReplaceRoundabouts::replaceRoundabouts(boost::shared_ptr<OsmMap> pMap)
 
   // Get a list of roundabouts from the map, go through & process them
   std::vector<RoundaboutPtr> roundabouts = pMap->getRoundabouts();
-
   for (size_t i = 0; i < roundabouts.size(); i++)
   {
     RoundaboutPtr pRoundabout = roundabouts[i];
     pRoundabout->replaceRoundabout(pMap);
     _numAffected++;
   }
+
+  //  Clean up any roundabout centers that didn't clean themselves up earlier
+  std::vector<long> centers = FindNodesVisitor::findNodesByTag(pMap, MetadataTags::HootSpecial(), "roundabout_center");
+  foreach (long id, centers)
+    RemoveNodeOp::removeNode(pMap, id, true);
```

Comment:
Some roundabout centers aren't cleaned up like they should be.  At this point there shouldn't be any left because the replacing of roundabouts is completed.  They should all be deleted.